update-imported-docs/reference.yml

```diff
@@ -8,13 +8,17 @@ repos:
     cd $GOPATH
     git clone https://github.com/kubernetes/kubernetes.git src/k8s.io/kubernetes
     cd src/k8s.io/kubernetes
-    git checkout release-1.16
+    git checkout release-1.17
     make generated_files
     cp -L -R vendor $GOPATH/src
-    rm -r vendor
+    rm -r vendor/github.com/spf13/cobra
+    rm -r vendor/github.com/spf13/pflag
+    #rm -r vendor
```

Comment:
Emm. I tried the `make` and encountered the same error you mentioned somewhere else. I solved it by removing vendor/golang.in/ ... packages. It is annoying that Go is changing is behavior in an incompatible way.

Reply:
I'm not sure what has changed, but I upgraded to:
go version go1.13.5 linux/amd64
The original vendor issue went away, except for the two libraries listed above.
I can test again tomorrow.

Reply:
Tested again. Lines 14-15 appear to be all that is required (for me).